### PR TITLE
First plain english review 

### PR DESCRIPTION
**Pull request by lrsantos11**
Regards #32  and #38, however not entirely.

### CODE REVIEW

File: overview.tex [hunk 1]
```diff
@@ -122,25 +124,26 @@ \subsection*{Implementation and architecture}
       \item {\tt perprof/main.py} that has the command line interface.
     \end{itemize}
 
-    The choice for perprof-py not be compatible with Python 2
-    was due (i) the fact that unicode processing with Python 2 can be a nightmare and
-    (ii) the authors desire to push Python 3 forward.
+    Incompatibility of perprof-py  with Python 2
+    was due (i) to the fact that unicode processing with Python 2 can be a
+    nightmare, and
+    (ii) to the authors desire to push Python 3 forward.
```
Comment (rgaiacs) [inline overview.tex]: "the authors desire" -> "he authors' desire"?

File: overview.tex [hunk 2]
```diff
@@ -183,20 +186,20 @@ \subsection*{Input}
...
+    Also, using  both YAML or command line options,  user can change each
+    columns meaning.
     Notice that these options are not enabled by default. The user should
-    consult the help and documentation to see how to enable them.
+    consult  help and documentation files to see how to enable them.
 
 \subsection*{Parsing process and output}
 
-    To use perprof-py, the user needs to issue a command of the type
+    To use perprof-py, tone needs to issue a command of the type
```
Comment (rgaiacs) [inline overview.tex]: Is the first time that I read "tone" as use here. =)

**Comment by rgaiacs** [comment]
@lrsantos11 Do you prefer that we merge this and later you open another pull request or keep this active so that you could add more commits on it?

**Comment by lrsantos11** [comment]
I think is better to keep this active and I update your suggestions

File: overview.tex (hunk 2 above)
Comment (lrsantos11) [inline overview.tex]: LOL

File: overview.tex [hunk 3]
```diff
@@ -85,26 +85,28 @@ \subsection*{Motivation}
...
-    proprietary solutions, made their own implementation and then made them
-    available, such as a Python function perfprof from \textcite{url:perfprof} 
-    and a small Julia module {\tt perfprof.jl} from \textcite{url:perfprofjl}, 
+    Some, perhaps unaware of these implementations or trying to avoid
+    proprietary solutions, implemented their own solutions and then made
+    them
+    available. Solutions such as a Python function perfprof from \textcite
+    {url:perfprof}
+    and a  Julia module {\tt perfprof.jl} from \textcite{url:perfprofjl}, 
     both language dependent. A through search would possible reveal many
```
Comment (abelsiqueira) [inline overview.tex]: "possibly"

File: overview.tex [hunk 4]
```diff
@@ -183,20 +186,20 @@ \subsection*{Input}
...
+    These options were set from {\tt algname}, {\tt success}, and {\tt
     free\_format} options, respectively.
 
-    The user can, optionally, add more columns to give additional information.
+    A user can, optionally, add more columns to add information.
     He can verify, for instance, that the optimality conditions are satisfied
     for each problem.
-    Also, using  YAML or  command line options, the user can change the
-    meaning of each columns.
+    Also, using  both YAML or command line options,  user can change each
```
Comment (abelsiqueira) [inline overview.tex]: "both" -> "either"?

File: overview.tex [hunk 5]
```diff
@@ -183,20 +186,20 @@ \subsection*{Input}
...
+    columns meaning.
```
Comment (abelsiqueira) [inline overview.tex]: columns'

**Comment by lrsantos11** [comment]
My final revision will be submitted on the end of this afternoon. Please, if you want, give the last review so we can send to JORS today.

**Comment by rgaiacs** [comment]
> My final revision will be submitted on the end of this afternoon.

OK

> Please, if you want, give the last review so we can send to JORS today.

https://github.com/lrsantos11/perprof-jors/pull/1

**Comment by lrsantos11** [comment]
@rgaiacs Didn't get the last comment.

**Comment by rgaiacs** [comment]
@lrsantos11 After you merge this, +1 to send to JORS.

**Comment by abelsiqueira** [comment]
I'm doing my review now, but just occurred to me: Do we need a new release?

**Comment by abelsiqueira** [comment]
lrsantos11#2 are my suggestions.

**Comment by abelsiqueira** [comment]
I created the new release information #76. If you want, we can merge this first, then I rebase.
After that, :+1: to submit.

**Comment by lrsantos11** [comment]
Guys, I guess is done.

**Comment by abelsiqueira** [comment]
Thanks

**Pull request merged**